util/log/src/Makefile

```diff
@@ -0,0 +1,7 @@
+log:log_yadl.o liblog.a
```

Comment:
1) The Makefile shouldnot be created by you. You need to create Makefile.am instead, which autogen.sh picks up and creates the Makefile for you.

 2) You need an entry in configure.ac for util/log/src/